haystack/components/readers/extractive.py

```diff
@@ -42,7 +42,8 @@ class ExtractiveReader:
     reader.warm_up()
 
     question = "Who lives in Berlin?"
-    reader.run(query=question, documents=docs)
+    result = reader.run(query=question, documents=docs)
+    result["answers"]
```

Comment:
nitpick: maybe add the output here, or better since there's nothing to show maybe just:

```suggestion
    reader.run(query=question, documents=docs)
 ```